styles/tools/_functions.scss

```diff
@@ -0,0 +1,5 @@
+@import "./../settings/colors";
```

Comment:
Let's try importing the colors directly on to the tests instead of in the functions file. We need the colors to test this function, but the function itself doesn't need the context of the color map.

Reply:
Taken care of.